package.json

```diff
@@ -40,5 +43,30 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "eslint-config-prettier": "^7.2.0",
+    "eslint-plugin-prettier": "^3.3.1",
+    "husky": "^4.3.8",
+    "lint-staged": "^10.5.4",
+    "prettier": "^2.2.1",
+    "pretty-quick": "^3.1.0",
+    "ts-jest": "^26.5.0"
+  },
+  "jest": {
+    "transform": {
+      "^.+\\.ts?$": "ts-jest"
+    },
+    "transformIgnorePatterns": [
+      "node_modules/(?!(jsoneditor-react))/"
+    ]
+  },
+  "lint-staged": {
+    "*.js": "eslint"
+  },
+  "husky": {
```

Comment:
This would be the hooks to use for pre-commits. Potentially adding the tests here.